alternator/executor.cc

```diff
@@ -664,12 +664,48 @@ static future<bool> is_view_built(
 
 }
 
+void executor::notify_all_shards_of_newly_calculated_table_size(schema_ptr schema, std::uint64_t size_in_bytes, std::chrono::nanoseconds ttl) {
+    auto &table = schema->table();
+    auto &describe_table_info = _describe_table_info_for_tables[&table];
```

Comment:
You said you changed the cache index from a pointer (!?) to a table ID, but I don't see this change here.
But I'm worried - I don't understand where we limit the size of this cache. If you create a workload which creates and delete table a million times in a loop (try it :-)), you'll end up caching the size (0) for a million already-deleted tables, and these cached values of deleted tables will never be clearer. That's not great.